test/Spring-test.js

```diff
@@ -351,6 +351,9 @@ describe('Spring', () => {
       },
 
       componentDidMount() {
+        this.update();
```

Comment:
On second/third thought, let's keep it this way. I'm pretty sure I'd be responsible with it if I ever need it, but let's dissuade potential contributors from using this pattern.